recipes/starshot_card_block_type/config/core.entity_view_display.block_content.card.default.yml

```diff
@@ -0,0 +1,46 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - block_content.type.card
+    - field.field.block_content.card.body
+    - field.field.block_content.card.field_card_link
+    - field.field.block_content.card.field_card_media
+  module:
+    - link
+    - text
+id: block_content.card.default
+targetEntityType: block_content
+bundle: card
+mode: default
+content:
+  body:
+    type: text_default
+    label: hidden
+    settings: {  }
+    third_party_settings: {  }
+    weight: 0
+    region: content
+  field_card_link:
+    type: link
+    label: above
```

Comment:
I assume this wasn't intentional

Reply:
Good catch. Fixed in 6808e62dfbebfbb2fa10b020e5a09e3d0dd5dc81.